app/controllers/meetings_controller.rb

```diff
@@ -450,5 +359,49 @@ def hide_page(meeting)
     )
     !(meeting_obj.exists? && meeting_member.exists?)
   end
+
+  # rubocop:disable MethodLength
+  def notifications_for_members(members, type)
```

Comment:
I think it would be a good idea to move this out of the controller and into a separate `Notifications` service.

Reply:
That's a better idea, I'll look into that. Thanks!

Reply:
I should as well. I've seen it done, but never written it myself.